.github/workflows/release-please.yml

```diff
@@ -1,9 +1,39 @@
+# Validates Functions Framework at a specific commit and uses the
+# Release Please GitHub Action to create a PR that can be approved & merged
+# to generate a GitHub Release.
+name: Create Release Candidate
 on:
   push:
     branches:
       - main
+  workflow_dispatch:
 jobs:
+  go113-buildpack-test:
+    uses: GoogleCloudPlatform/functions-framework-conformance/.github/workflows/buildpack-integration-test.yml@v1.4.0
+    with:
+      http-builder-source: 'testdata/conformance/function'
+      http-builder-target: 'declarativeHTTP'
+      cloudevent-builder-source: 'testdata/conformance/function'
+      cloudevent-builder-target: 'declarativeCloudEvent'
+      prerun: 'testdata/conformance/function/prerun.sh ${{ github.sha }}'
+      builder-runtime: 'go113'
+      builder-tag: 'go113_20220320_1_13_15_RC00'
```

Comment:
Perhaps add a note here as to why we're using this builder / how we will update this in the future.